### DISCUSSION TITLE
Logo display

### DISCUSSION THREAD

**Discussion by easybeau** [mmistakes/jekyll-theme-basically-basic#152] [Q&A] [answered]
Hello, first of all, thanks for this beautiful theme.

## Environment

- Basically Basic version: latest
- Ruby gem or remote theme version: ruby gem
- Jekyll version: jekyll 4.2.1
- Git repository URL: https://github.com/easybeau/vibe-friperie
- GitHub Pages hosted (if yes provide URL to site):
- Operating system: Pardus GNU/Linux 21 based on debian

## Steps to reproduce the behavior

I created a folder img in assets and put logo.png in it. then I added this line -> logo : assets/img/logo.png on _config.yml.
My logo is 260x300px and I'd like to display the full size on my website without the site title.
![loggo](https://user-images.githubusercontent.com/7128391/143430747-d491ce87-2e2e-4071-981f-718f98075120.png)

Thanks for your help,

Best regards,
Isabelle

**Comment by mmistakes** [comment, marked as answer]
You'll need to [customize the theme's CSS](https://github.com/mmistakes/jekyll-theme-basically-basic#customizing-sass-scss).
Currently `.site-logo` sets the image to 50x50 pixels in a circle.

You can change the `width` and `height` in [this Sass partial](https://github.com/mmistakes/jekyll-theme-basically-basic/blob/master/_sass/basically-basic/_global.scss#L37-L42) and remove `border-radius: 50%` to get rid of the rounded corners.

**Reply by easybeau** [reply]
Thank you !
